### PR TITLE
Fix 'show-navbar' functionality

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

Fix 'show-navbar' functionality.

Screenshot from 2020-07-01 15-52-02
Screenshot from 2020-07-01 15-52-21

### CODE REVIEW

File: src/components/app-nav-menu/nav-menu.styl [hunk 1]
```diff
@@ -51,23 +51,56 @@ i.codicon-symbol-method
...
-    min-height 60px
-    max-height 60px
+    min-height 57px
+    max-height 57px
   100%
     overflow hidden
     min-height 0
     max-height 0
+
+.ui-container .pf-c-page
```
Comment (akurinnoy) [inline src/components/app-nav-menu/nav-menu.styl]: What was the problem you fixed by changing `.pf-c-page` background?

File: src/components/app-nav-menu/NavMenu.tsx [hunk 2]
```diff
@@ -167,15 +174,14 @@ export class NavMenu extends React.PureComponent<any, any> {
...
         logo={Logo}
         logoProps={{ href: 'https://www.eclipse.org/che/', target: '_blank' }}
         headerTools={headerTools}
-        showNavToggle={true}
-        onNavToggle={this.onNavToggle}
       />
     );
 
-    const Sidebar = <PageSidebar nav={PageNav} theme={theme} />;
+    const Sidebar = <PageSidebar nav={PageNav} theme={theme} className={isNavOpen ? 'sidebar-show' : 'sidebar-hide'} />;
```
Comment (akurinnoy) [inline src/components/app-nav-menu/NavMenu.tsx]: why don't you use `isNavOpen` property to set the sitebar visibility?

Comment (olexii4) [reply src/components/app-nav-menu/NavMenu.tsx:191]: It doesn't work in this way after the upgrade. We can read this variable but cannot set it.

But I will be glad to see an alternative solution.

File: src/components/app-nav-menu/nav-menu.styl (hunk 1 above)
Comment (olexii4) [reply src/components/app-nav-menu/nav-menu.styl]: Fixed

**Review by akurinnoy — APPROVED** [review]

**Pull request merged**